CHIPageControl/CHIPageControlAji.swift

```diff
@@ -105,4 +105,17 @@ open class CHIPageControlAji: CHIBasePageControl {
         return CGSize(width: CGFloat(inactive.count) * self.diameter + CGFloat(inactive.count - 1) * self.padding,
                       height: self.diameter)
     }
+    
+    override open func didTouch(gesture: UITapGestureRecognizer) {
+        var touchIndex: Int?
+        let point = gesture.location(ofTouch: 0, in: self)
+        inactive.enumerated().forEach({ count, layer in
```

Comment:
I think this way would be more compact and it avoids force casting. 
```
if let touchIndex = inactive.enumerated().first(where: { $0.element.hitTest(point) != nil })?.offset {
    delegate?.didTouch(pager: self, index: touchIndex)
}
```

Can be changed for all other page controls as well.

Reply:
Great, I just updated my repo with this change.